### PR TITLE
wgMinervaEnableSiteNotice  on ManageWiki per T5057

### PR DESCRIPTION
**Pull request by Hispano76**
It needs thorough revision! It may have errors.

### CODE REVIEW

**Comment by paladox** [comment]
Need to also set wgMinervaEnableSiteNotice in LocalSettings

File: LocalSettings.php [hunk 1]
```diff
@@ -2080,6 +2080,10 @@
 		],
 	],
 
+	'wgMinervaEnableSiteNotice' => [
+		'default' => true,
```
Comment (paladox) [inline LocalSettings.php]: Should be set to false

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -1338,6 +1338,15 @@
 		'section' => 'styling',
 		'help' => 'If not set, then the img.line element (white divider line in the top menu, between the site name and the down arrow/"Upload file" link) won\'t be generated.',
 	],
+	'wgMinervaEnableSiteNotice ' => [
+		'name' => 'Minerva Sitenotice',
+		'from' => 'minerva',
```
Comment (paladox) [inline ManageWikiSettings.php]: mobilefrontend

File: LocalSettings.php (hunk 1 above)
Comment (Hispano76) [reply LocalSettings.php]: check

File: ManageWikiSettings.php (hunk 2 above)
Comment (Hispano76) [reply ManageWikiSettings.php]: check

File: ManageWikiSettings.php [hunk 3]
```diff
@@ -1338,6 +1338,15 @@
 		'section' => 'styling',
 		'help' => 'If not set, then the img.line element (white divider line in the top menu, between the site name and the down arrow/"Upload file" link) won\'t be generated.',
 	],
+	'wgMinervaEnableSiteNotice ' => [
+		'name' => 'Minerva Sitenotice',
+		'from' => 'mobilefrontend',
+		'restricted' => false,
+		'type' => 'check',
+		'overridedefault' => true,
```
Comment (Hispano76) [inline ManageWikiSettings.php]: Um, that should be in false, right? 'Cause you check the box in ManageWiki.

Comment (paladox) [reply ManageWikiSettings.php]: Nope, this makes sure it is true by default, allowing users to unset it.

If you set this as false it'll mean it defaults to false.

Comment (Hispano76) [reply ManageWikiSettings.php]: Okay, I wanted to make sure. Thank you.

**Pull request merged**